tests/src/rules/newline-after-import.js

```diff
@@ -852,14 +861,26 @@ ruleTester.run('newline-after-import', require('rules/newline-after-import'), {
       parserOptions: { ecmaVersion: 2015, considerComments: true, sourceType: 'module' },
     },
     {
-      code: `const foo = require('foo');\n\n\n// some random comment\nconst bar = function() {};`,
-      options: [{ count: 2, exactCount: true, considerComments: true }],
```

Comment:
can we keep this existing test case, to ensure it's not a breaking change?

Reply:
This test case is valid, so if we want to keep it, I think the test case should be included in the valid array. Is it okay to do this?

Reply:
Sure, that's fine - moving the test shows it's a bug fix.

Reply:
thanks, I applied it.